.github/workflows/testing.yml

```diff
@@ -81,9 +84,15 @@ jobs:
         run: |
           tt init
           sudo tt install tarantool master
+          
+      - name: Setup Tarantool from PR
+        if: matrix.tarantool == 'pr/8630'
+        run: |
+          tt init
+          tt install tarantool pr/8630
```

Comment:
2.x latest setup uses `sudo install`, you may need this too.

Reply:
The workflow succeeded, looks like `sudo` is unnecessary.